### ISSUE TITLE
Building model

### ISSUE THREAD

**Issue by msaakaash** [AnikethaPrasad/Multivariate-Stock-Price-Forecasting#6]
## 🚀Begin the development of the LSTM-model:

- Load the train,test,validate data 
- Define the LSTM architecture using TensorFlow/Keras.
- Compile and set up loss functions, optimizers, and evaluation metrics.
